### ISSUE TITLE
Memory plugin not reporting percentage

### ISSUE THREAD

**Issue by jeffastorey** [collectd/collectd#1476]
I've configured the memory plugin to send out memory percentage stats. In my collectd conf file I have:

    LoadPluginMemory
    <Plugin "memory">
      ValuesAbsolute "false"
      ValuesPercentage "true"
    </Plugin>

I enabled the CSV plugin and the memory used is showing as

    1450464406.032,86409216.000000

It always seems to output an absolute value rather than a percentage.

**Comment by rubenk** [comment]
Can you try with `ValuesPercentage true`, so without the quotes?

**Comment by jeffastorey** [comment]
Yep, that did it. Thanks.

**Comment by rubenk** [comment]
Ok, good to hear!

**Comment by ecentinela** [comment]
I'm having the same problem with memory and df plugins. My configuration is:

```
LoadPlugin df

<Plugin df>
    Device "/dev/xvda1"
    ValuesAbsolute false
    ValuesPercentage true
</Plugin>

LoadPlugin memory

<Plugin memory>
    ValuesAbsolute false
    ValuesPercentage true
</Plugin>
```

And I'm getting results as 79,757,312 for memory free.

Did you know what I'm doing wrong?

**Comment by octo** [comment]
Hi @ecentinela, unfortunately I cannot reproduce the problem with the current *master* branch. Have you tried upgrading to either 5.4.3 or 5.5.2?

**Comment by ecentinela** [comment]
Fixed in 5.5!
